packages/react-components/theme-designer/src/ThemeDesigner.tsx

```diff
@@ -1,11 +1,21 @@
 import * as React from 'react';
 import { makeStyles } from '@griffel/react';
 import type { ThemeDesignerProps } from './ThemeDesigner.types';
-import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
-import { createLightTheme, createDarkTheme } from '@fluentui/react-theme';
+import { FluentProvider } from '@fluentui/react-components';
+import {
+  createLightTheme,
+  createDarkTheme,
+  teamsLightTheme,
+  teamsDarkTheme,
+  webLightTheme,
+  webDarkTheme,
+  BrandVariants,
+  Theme,
```

Comment:
_nit:_ Order these alphabetically

```suggestion
  createDarkTheme,
  createLightTheme,
  teamsDarkTheme,
  teamsLightTheme,
  webDarkTheme,
  webLightTheme,
  BrandVariants,
  Theme,
```

Reply:
Also, you could split `BrandVariants` and `Theme` into a separate `import type` statement

`import type { BrandVariants, Theme } from '@fluentui/react-theme'`

Reply:
Also, I think you can technically import all of these directly from the suite `@fluentui/react-components` package instead of leveraging the individual `@fluentui/react-theme` package, since it seems you're already importing things from the former anyway.